### ISSUE TITLE
Feedback on v3 with reledmac

### ISSUE THREAD

**Issue by maieul** [FrankMittelbach/fmitex#1]
Dear Frank, I tried the master branch with reledmac/reledpar handbook. I need to do some corrections on my .dtx
https://github.com/maieul/ledmac/commit/e5b14fe2010de950eef6befd0d8ec124512d95e9

- Package options clashes for hyperref (in my case, I don't think it matter any thing)
-\SpecialMainIndex command  does not exist any more (I don't need it, it was added by Peter Wilson)
- When I do my "make reledmac.pdf", on the second xelatex run (so after makeglossaris run), I get

```
Text for \verb command ended by end of line.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.489 ... deleted in version 0.17|hdpindex{}\pfill
```

Which I didn't get in the past. The problem line is `subitem \verb*+\print@line+:\ Coming back of add@penalties, mistakenly deleted in version 0.17|hdpindex{}\pfill` in reledmac.glo

**Comment by FrankMittelbach** [comment]
@maieul thanks for the first test. 

 - hyperref: I guess that is one of the possible candidates for failure, but on the whole (may have to change my mind about this) I think that it is correct for doc to load it by default as that is what most often will be needed. You can prevent that through an option to doc now.
 - `\SpecialMainIndex`: this is an oversight by me, that should still be there, there is even en entry saying "describe old ...` but somehow it got lost. Will readd that.
 - as to the `\verb` error I'm not sure I already understand that (probably have to look at our file directly) --- are you saying this is a bug in the new doc.sty or a problem in your file  that now shows up with the new doc?

**Comment by maieul** [comment]
1. Hyperref > yes, that is a good idea. I just tell you, but I think that must be by default.

**Comment by maieul** [comment]
2. SpecialMainIndex -> so I will wait.

**Comment by maieul** [comment]
3. Verb -> I need more investigation. I could do it Friday, and tell you if the problem is on my code or on yours

**Comment by FrankMittelbach** [comment]
Am 17.04.18 um 20:24 schrieb Maïeul Rouquette:
> Verb -> I need more investigation. I could do it Friday, and tell you if 
> the problem is on my code or on yours

ok will wait for this then ... have a hard disk failure on my desktop 
and currently busy replacing disks and restoring backups anyway ... good 
to have daily backups :-)

**Comment by FrankMittelbach** [comment]
Am 17.04.18 um 20:23 schrieb Maïeul Rouquette:
> SpecialMainIndex -> so I will wait.

added

**Comment by FrankMittelbach** [comment]
Am 17.04.18 um 20:23 schrieb Maïeul Rouquette:
> Hyperref > yes, that is a good idea. I just tell you, but I think that 
> must be by default.

on the whole this needs some further thoughts how to best handle that as 
that particular problem will probably show up in other circumstances too 
(but not tonight)

**Comment by maieul** [comment]
Concerning the verb problem, https://github.com/maieul/ledmac/commit/16c86b96f71fa7ac9177803b9268989947f0208b  solve it. I don't know if the problem was mine or yours.

**Comment by FrankMittelbach** [comment]
thanks. The \SpecialMainIndex should have been resolved with 2780f42dedda36728ec5f470b929a48fcb878d57 so that what now remains is deciding how to best handled hyperref option clashes
